taxi/forms.py

```diff
@@ -0,0 +1,54 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.forms import (
+    ModelForm,
+    CheckboxSelectMultiple,
+    ModelMultipleChoiceField
+)
+
+from .models import Driver, Car
```

Comment:
It is a better choice to use absolute imports instead of relative ones in Django projects to not depend on the file placement.